docker-images/base/Dockerfile

```diff
@@ -9,7 +9,7 @@ ARG TARGETARCH
 USER root
 
 RUN microdnf update \
```

Comment:
Should we also add `-y` here to avoid the warning?

Reply:
Unlike dnf, looks like microdnf does not require this flag by default, as it was designed to build containers. Still there is some build tool may raise a warning, even if it shouldn't. I thought you added the flag because of that, or?

Reply:
I added the `-y` flags to places where the build on UBI9 got stuck without them waiting for a user configuration (they were not needed in UBI8). But I never saw any warnings related to that and they were not my motivation for the flags.

Reply:
Ok, makes sense. Is it not required for the update command?

Reply:
No, doesn't seem to.

Reply:
Ok, that's consistent with dnf behavior.